dotnet/src/SemanticKernel/Kernel.cs

```diff
@@ -83,67 +88,76 @@ public sealed class Kernel : IKernel, IDisposable
         this._memory = memory;
         this._aiServiceProvider = aiServiceProvider;
         this._promptTemplateEngine = promptTemplateEngine;
-        this._skillCollection = skillCollection;
+        this._functionCollection = functionCollection;
 
         this._logger = loggerFactory.CreateLogger(typeof(Kernel));
     }
 
     /// <inheritdoc/>
     public ISKFunction RegisterSemanticFunction(string functionName, SemanticFunctionConfig functionConfig)
     {
-        return this.RegisterSemanticFunction(SkillCollection.GlobalSkill, functionName, functionConfig);
+        return this.RegisterSemanticFunction(FunctionCollection.GlobalFunctions, functionName, functionConfig);
     }
 
     /// <inheritdoc/>
-    public ISKFunction RegisterSemanticFunction(string skillName, string functionName, SemanticFunctionConfig functionConfig)
+    public ISKFunction RegisterSemanticFunction(string pluginName, string functionName, SemanticFunctionConfig functionConfig)
     {
         // Future-proofing the name not to contain special chars
-        Verify.ValidSkillName(skillName);
+        Verify.ValidPluginName(pluginName);
         Verify.ValidFunctionName(functionName);
 
-        ISKFunction function = this.CreateSemanticFunction(skillName, functionName, functionConfig);
-        this._skillCollection.AddFunction(function);
+        ISKFunction function = this.CreateSemanticFunction(pluginName, functionName, functionConfig);
+        this._functionCollection.AddFunction(function);
 
         return function;
     }
 
     /// <inheritdoc/>
-    public IDictionary<string, ISKFunction> ImportSkill(object skillInstance, string? skillName = null)
```

Comment:
Some of these are on the critical path for callers... can you restore `ImportSkill` as obsolete and have it call the `ImportPlugin`?

Reply:
Yes, that's already in place. All of the main methods/properties that I changed have an original stub which calls the new method/property.